Spec/RealtimeClientPresence.swift

```diff
@@ -3814,6 +3844,47 @@ class RealtimeClientPresence: QuickSpec {
                     expect(decodeNumberOfCalls).to(equal(1))
                 }
 
+                // RTP14d
+                it("should be present all the registered members on a presence channel") {
+                    let client = ARTRealtime(options: AblyTests.commonAppSetup())
+                    defer { client.dispose(); client.close() }
+                    let channel = client.channels.get("foo")
+
+                    let john = "john"
+                    let max = "max"
+
+                    waitUntil(timeout: testTimeout) { done in
+                        let partialDone = AblyTests.splitDone(4, done: done)
+                        channel.presence.enterClient(john, data: nil) { error in
+                            expect(error).to(beNil())
+                            partialDone()
+                        }
+                        channel.presence.enterClient(max, data: nil) { error in
+                            expect(error).to(beNil())
+                            partialDone()
+                        }
+                        channel.presence.subscribe { message in
```

Comment:
Shouldn't this subscription be done before the `enterClient`s, so that we make sure we don't miss them?